src/redis_cmd.cc

```diff
@@ -1453,6 +1453,48 @@ class CommandHGetAll : public Commander {
   }
 };
 
+class CommandHRange : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    if (args.size() != 6 && args.size() != 4) {
+      return Status(Status::RedisParseErr, errWrongNumOfArguments);
+    }
+    if (args.size() == 6 && Util::ToLower(args[4]) != "limit") {
+      return Status(Status::RedisInvalidCmd, errInvalidSyntax);
+    }
+    try {
+      start_ = std::stol(args[2]);
+      stop_ = std::stol(args[3]);
+      if (args.size() == 6) {
+        limit_ = std::stol(args[5]);
```

Comment:
Can use `ParseInt` here

Reply:
> Can use `ParseInt` here

I noticed that std::stol is used in quite a few places in the kvrocks. Should I open another issue to replace these std::stol?

Reply:
Yes, I think we can do it in another PR to make the context clear.